src/plugins/intel_cpu/thirdparty/ACLConfig.cmake

```diff
@@ -335,8 +335,6 @@ elseif(NOT TARGET arm_compute::arm_compute)
         set(arm_compute_full_path "${arm_compute}")
     endif()
 
-    list(APPEND ARM_COMPUTE_OPTIONS experimental_fixed_format_kernels=True)
```

Comment:
Why it was deleted? Is it default value in ACL now?

Reply:
This param was deleted and now build is failed with it